notebooks/LST1_observation_simulator.ipynb

```diff
@@ -0,0 +1,1013 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    #     ebl_file = subprocess.run(['find', str(gammapy_dir), '-name', 'ebl_dominguez11.fits.gz'], 
</pre><p>I think this can be removed, and simply leave a comment saying that if you have GAMMAPY_DATA env variable set, you can also use the builtin models. But this find subprocess will probably not work for all users</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Added some comments. I left there the commented code, since it works at least in some cases.